### DISCUSSION TITLE
Removing default folder icon

### DISCUSSION THREAD

**Discussion by podloznyi** [retypeapp/retype#467] [Q&A] [answered]
Hello folks, is there anyway to remove/disable default folder icon?
![image](https://user-images.githubusercontent.com/60586595/210016028-42aef7ae-ac6a-4857-9f14-00477bfb13f1.png)

**Comment by geoffreymcgill** [comment, marked as answer]
Hello. If you have a `default.md`, `index.md`, or `readme.md` in that folder, then you can set the `icon` config to any other icon value. For example, setting to `icon: dot`. 

#### default.md

```md
---
icon: dot
---
# Sample page
```

<img width="287" alt="Screenshot 2022-12-29 at 11 35 05 PM" src="https://user-images.githubusercontent.com/62210/210017537-378809fa-740d-46cd-a35f-9f08b83e9fd2.png">


If you do not have a `default.md`, `index.md`, or `readme.md` file in the folder, you can add a `index.yml` file into the folder and that will allow for the same customization of the folder properties. 

#### index.yml

```yml
icon: dot
```

At the moment, Retype does not have a way to completely remove the icon, although you can work-around that limitation by creating a transparent `.png` and linking to the that image. The following sample demonstrates setting the `icon` to a custom transparent `.png` file that you can add anywhere within your project.

#### index.yml

```yml
icon: "/static/blank.png"
```

Hope this helps.

**Reply by podloznyi** [reply]
Trick with .png worked for me, thanks!
